actions/core.php

```diff
@@ -317,16 +317,18 @@ public function donors_group_user_add()
 		$payer_id = (int) $this->payer_data['user_id'];
 		$payer_username = $this->payer_data['username'];
 		$default_group = $this->config['ppde_ipn_group_as_default'];
+		$payer_donated_amount = $this->payer_data['user_ppde_donated_amount'];
 
 		/**
 		 * Event to modify data before a user is added to the donors group
 		 *
 		 * @event skouat.ppde.donors_group_user_add_before
-		 * @var bool    can_use_autogroup   Whether or not to add the user to the group
-		 * @var int     group_id            The ID of the group to which the user will be added
-		 * @var int     payer_id            The ID of the user who will we added to the group
-		 * @var string  payer_username      The user name
-		 * @var bool    default_group       Whether or not the group should be made default for the user
+		 * @var bool    can_use_autogroup      Whether or not to add the user to the group
+		 * @var int     group_id               The ID of the group to which the user will be added
+		 * @var int     payer_id               The ID of the user who will we added to the group
+		 * @var string  payer_username         The user name
+		 * @var bool    default_group          Whether or not the group should be made default for the user
+		 * @var float   payer_donated_amount   The user donated amount
 		 * @since 1.0.3
```

Comment:
Please, after the `since` comment add a `change` comment like this example: https://github.com/phpbb/phpbb/blob/3.2.x/phpBB/includes/message_parser.php#L1222

Reply:
Ok.

Reply:
PPDE version will be : 2.1.2 :)

Reply:
Ok, I'll add that too.